manifest.toml

```diff
@@ -0,0 +1,64 @@
+packaging_format = 2
+
+id = "gitea"
+name = "Gitea"
+description.en = "Lightweight self-hosted Git forge"
+description.fr = "Forge Git légère auto-hébergé"
+
+version = "1.18.5~ynh1"
+
+maintainers = ["Josué Tille"]
+
+[upstream]
+license = "MIT"
+website = "https://gitea.io/"
+demo = "https://try.gitea.io/"
+admindoc = "https://docs.gitea.io/en-us/"
+code = "https://github.com/go-gitea/gitea"
+
+[integration]
+yunohost = ">= 11.1.11"
+architectures = "all"
+multi_instance = true
+ldap = true
+sso = false
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+
+    [install.path]
+    type = "path"
+    default = "/gitea"
+
+    [install.admin]
+    type = "user"
```

Comment:
I'm not sure this question is still useful now if we go with `admin.allowed = ["admins"]` below ...

Or another way is to have a question named `init_admin_permission` and remove the `admin.allowed` line (but then we still need something to declare that the admin permission does exists